Gemfile.lock

```diff
@@ -7,41 +7,73 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    diff-lcs (1.1.3)
-    ffi (1.1.5)
-    guard (1.4.0)
-      listen (>= 0.4.2)
-      thor (>= 0.14.6)
-    guard-rspec (2.1.0)
-      guard (>= 1.1)
-      rspec (~> 2.11)
-    listen (0.5.3)
-    mock_redis (0.5.4)
-    rake (0.9.2.2)
-    rb-fsevent (0.9.2)
-    rb-inotify (0.8.8)
-      ffi (>= 0.5.0)
-    redis (3.2.0)
-    rspec (2.11.0)
-      rspec-core (~> 2.11.0)
-      rspec-expectations (~> 2.11.0)
-      rspec-mocks (~> 2.11.0)
-    rspec-core (2.11.1)
-    rspec-expectations (2.11.3)
-      diff-lcs (~> 1.1.3)
-    rspec-mocks (2.11.3)
-    thor (0.16.0)
-    timecop (0.5.2)
+    coderay (1.1.3)
+    diff-lcs (1.5.0)
+    ffi (1.15.5)
+    formatador (1.1.0)
+    guard (2.18.0)
+      formatador (>= 0.2.4)
+      listen (>= 2.7, < 4.0)
+      lumberjack (>= 1.0.12, < 2.0)
+      nenv (~> 0.1)
+      notiffany (~> 0.0)
+      pry (>= 0.13.0)
+      shellany (~> 0.0)
+      thor (>= 0.18.1)
+    guard-compat (1.2.1)
+    guard-rspec (4.7.3)
+      guard (~> 2.1)
+      guard-compat (~> 1.1)
+      rspec (>= 2.99.0, < 4.0)
+    listen (3.7.1)
+      rb-fsevent (~> 0.10, >= 0.10.3)
+      rb-inotify (~> 0.9, >= 0.9.10)
+    lumberjack (1.2.8)
+    method_source (1.0.0)
+    mock_redis (0.31.0)
+      ruby2_keywords
+    nenv (0.3.0)
+    notiffany (0.1.3)
+      nenv (~> 0.1)
+      shellany (~> 0.0)
+    pry (0.14.1)
+      coderay (~> 1.1)
+      method_source (~> 1.0)
+    rake (13.0.6)
+    rb-fsevent (0.11.1)
+    rb-inotify (0.10.1)
+      ffi (~> 1.0)
+    redis (4.6.0)
+    rspec (3.11.0)
+      rspec-core (~> 3.11.0)
+      rspec-expectations (~> 3.11.0)
+      rspec-mocks (~> 3.11.0)
+    rspec-core (3.11.0)
+      rspec-support (~> 3.11.0)
+    rspec-expectations (3.11.0)
+      diff-lcs (>= 1.2.0, < 2.0)
+      rspec-support (~> 3.11.0)
+    rspec-mocks (3.11.1)
+      diff-lcs (>= 1.2.0, < 2.0)
+      rspec-support (~> 3.11.0)
+    rspec-support (3.11.0)
+    ruby2_keywords (0.0.5)
+    shellany (0.0.1)
+    thor (1.2.1)
+    timecop (0.9.5)
```

Comment:
### [timecop](https://github.com/travisjeffery/timecop)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 0.5.2 → 0.9.5

[_~~change-log~~, [source-code](https://github.com/travisjeffery/timecop), [gem-diff](https://my.diffend.io/gems/timecop/0.5.2/0.9.5)_]





<details>
<summary>Commits</summary>

A change of **278** commits. See the full changes on [the compare page](https://github.com/travisjeffery/timecop/compare/v0.5.2...v0.9.5).

These are the first 10 commits:
- (a7a98ac) [Merge pull request #69 from jordimassaguerpla/master](https://github.com/travisjeffery/timecop/commit/a7a98acd06f841ad551333c488c5416f90bac9b3)
- (0d6c58f) [fix typo](https://github.com/travisjeffery/timecop/commit/0d6c58fa20368d69fbcdbc075a9724b64acbafaf)
- (c3b466a) [Refactor tests so order doesn't matter.](https://github.com/travisjeffery/timecop/commit/c3b466a3bd2a0340a7e2425a0e149720c66c2563)
- (bf5d377) [ActiveSupport::TimeWithZone#to_r is broken, which](https://github.com/travisjeffery/timecop/commit/bf5d37798566e1edc616d5cbea87466517afac29)
- (da2943b) [Refactor initial parsing of frozen time, add a test](https://github.com/travisjeffery/timecop/commit/da2943bc66e034ed03a840455c6f5bf5b3c6cf9b)
- (0dc15bc) [Fix bug with timezones losing info on freeze and travel](https://github.com/travisjeffery/timecop/commit/0dc15bc0c2347008f7a0796ff38bec6e821ad012)
- (8c4df10) [Merge pull request #73 from tommeier/master](https://github.com/travisjeffery/timecop/commit/8c4df106f2fbe788d13171dc835d587d977a8b38)
- (b86f633) [v0.6.0](https://github.com/travisjeffery/timecop/commit/b86f633b44f6b818f63c36e939c449693482b95c)
- (ef849ff) [Check Time.zone isn't nil](https://github.com/travisjeffery/timecop/commit/ef849ff515b5619ee32e3675f48e06fd0601d410)
- (ff8b96e) [Merge pull request #74 from hamakn/check_time_zone](https://github.com/travisjeffery/timecop/commit/ff8b96e08a1ede8aa3bbea0ec5e8f22a9b982de9)


</details>